<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonPut">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Disconnect">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="IPText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="minSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="maxSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="timmingSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Timming</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>MIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>50</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>MAX</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minLCD">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="maxLCD">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="saidaTexto">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timmingLCD">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Connect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpConnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Disconnect</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>tcpDisconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>minLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>maxLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>204</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timmingSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>timmingLCD</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonStop</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timmingSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MainWindow</receiver>
   <slot>sliderStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getText()</slot>
  <slot>tcpConnect()</slot>
  <slot>tcpDisconnect()</slot>
  <slot>stop()</slot>
  <slot>slot2()</slot>
  <slot>start()</slot>
  <slot>sliderStart()</slot>
 </slots>
</ui>
